plugin/al/plugin/AL_USDMayaTestPlugin/py/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ separate_argument_list(pythonPath)
 separate_argument_list(mayaPluginPath)
 
 add_test(
-    NAME ${TEST_NAME}
+    NAME TestUSDMayaPython
```

Comment:
No need to revert this part of #198 where we switched to use the variable `TEST_NAME`.

Reply:
I reverted because I now need 2 tests (among other reasons, we can't bundle the new test in because it needs maya.initialize to NOT be called), and I wasn't sure how to handle having two different TEST_NAME variables.  ie, I could do:

```
set(MAIN_TESTS_NAME, TestUSDMayaPython)
set(MODULE_ONLY_TESTS_NAME, TestUSDMayaPythonModuleOnly)

add_test( NAME ${MAIN_TESTS_NAME})
...
add_test( NAME ${MODULE_ONLY_TESTS_NAME})
```

...but at that point, I wasn't sure if we were gaining much by using variables instead of the test names directly.

However, if you want to go that route, I don't have a strong preference.

Reply:
Two different variables sound good to me.

Reply:
K, will do.